website/docs/d/subnet.html.markdown

```diff
@@ -40,6 +40,7 @@ data "aci_subnet" "dev_subnet" {
 - `annotation` - (Optional) Annotation for object subnet.
 - `description` - (Optional) Description for object subnet.
 - `ctrl` - (Optional) The list of subnet control state. The control can be specific protocols applied to the subnet such as IGMP Snooping.
+- `ip_data_plane_learning` - (Optional) Knob to disable IP Dataplane Learning for Host(/32, /128) and for BD Subnet. Type - String.
```

Comment:
Same applies here as with the resource documentation but allowed values and default can be omitted for non required/optional input. Also for those attributes, "(Optional)" should be replaced with (Read-Only) and put under Argument Reference header. See https://github.com/CiscoDevNet/terraform-provider-aci/pull/1102/files as an example

Reply:
Gotcha, thanks for the example. Renamed "Optional" to "Read-Only".